project_euler/problem_80/sol1.py

```diff
@@ -0,0 +1,31 @@
+"""
+    Euler Problem : 80
```

Comment:
- Please remove the indentation, it's not needed for module level docstring
- Simplify by removing @ symbol